pkg/client/clientset/versioned/typed/triggers/v1alpha1/triggers_client.go

```diff
@@ -59,22 +59,22 @@ func NewForConfig(c *rest.Config) (*TriggersV1alpha1Client, error) {
 	if err != nil {
 		return nil, err
 	}
-	return &TriggersV1alpha1Client{client}, nil
+	return &TektonV1alpha1Client{client}, nil
 }
 
-// NewForConfigOrDie creates a new TriggersV1alpha1Client for the given config and
+// NewForConfigOrDie creates a new TektonV1alpha1Client for the given config and
 // panics if there is an error in the config.
-func NewForConfigOrDie(c *rest.Config) *TriggersV1alpha1Client {
+func NewForConfigOrDie(c *rest.Config) *TektonV1alpha1Client {
 	client, err := NewForConfig(c)
 	if err != nil {
 		panic(err)
 	}
 	return client
 }
 
-// New creates a new TriggersV1alpha1Client for the given RESTClient.
-func New(c rest.Interface) *TriggersV1alpha1Client {
-	return &TriggersV1alpha1Client{c}
+// New creates a new TektonV1alpha1Client for the given RESTClient.
```

Comment:
This is a good question, and I actually don't know if we can keep it as `TriggersV1alpha1Client` without changing the groupName. As far as I know, the code generation creates the client name from the `groupName` in [doc.go](https://github.com/tektoncd/triggers/pull/65/files#diff-09308a4d5d5552db274496bb7141df8aR22). Since we want the groupName to be `tekton.dev` (at least for now), I think that using `TektonV1alpha1Client` is the way to go.

Reply:
@iancoffey, @bobcatfish do you have any input regarding this name change from `TriggersV1alpha1Client` to `TektonV1alpha1Client`? ^